seatunnel-e2e/seatunnel-e2e-common/src/test/resources/junit-platform.properties

```diff
@@ -0,0 +1,20 @@
+################################################################################
```

Comment:
Is this file necessary?

Reply:
Yes, otherwise the Order on the class does not take effect

Reply:
I feel a little confused. The scope of this file and the test case for adding `order` are not in the same package. Why do they affect each other?

Reply:
This is the feature of Junit5. The test case depend on the module of e2e-common. The file takes effect as long as you configure junit-platform.properties in the resource directory under test.

Reply:
Oh. I got it.

Reply:
Reference:
https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-execution-order-classes.

<img width="1020" alt="image" src="https://github.com/apache/seatunnel/assets/18141150/430e7763-97ca-4e36-96b3-1ab398edd8e8">